app/vmalert/main.go

```diff
@@ -95,9 +95,14 @@ func main() {
 	buildinfo.Init()
 	logger.Init()
 
-	err := templates.Load(*ruleTemplatesPath, true)
+	eu, err := getExternalURL(*externalURL)
 	if err != nil {
-		logger.Fatalf("failed to parse %q: %s", *ruleTemplatesPath, err)
+		logger.Fatalf("failed to init external.url %q: %s", *externalURL, err)
+	}
+
+	err = templates.Load(*ruleTemplatesPath, *externalURL)
```

Comment:
Passing raw `*externalURL` here would produce different results to `getExternalURL `.
Why not passing `eu *url.URL` instead?

Reply:
Good point, fixed in https://github.com/VictoriaMetrics/VictoriaMetrics/pull/7773/commits/4f32b5a9cbd837881ca37bd7dc4f8a4271cae346.